<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="minInt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Мин. случ. знач.</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="mainTable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>521</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="typeOfSorting">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Пузырьком</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Быстрая</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Расчёска</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Гномья</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Обезьянья</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="numOfRows">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Кол-во строк</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randomFill">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rand</string>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numOfColumns">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Кол-во столбцов</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textOut">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>281</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxInt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Макс. случ. знач.</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>281</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="getMinButton">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getAvrButton">
       <property name="text">
        <string>Avr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getMaxButton">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
